src/Libraries/Utils/Binary.idr

```diff
@@ -488,3 +487,15 @@ hashFile fileName
     osEscape = if isWindows
       then id
       else escapeStringUnix
+
+export
+hashFile : String -> Core String
+hashFile fileName =
+  -- Note `popen` call won't fail if there's no `sha256sum` command
+  -- so we can't just catch `popen` error.
+  catch (hashFileWith "sha256sum" fileName)
+        (\err =>
+          catch (hashFileWith "gsha256sum" fileName)
+                -- When `gsha256sum` fails as well, return the error with `sha256sum`.
```

Comment:
Hey @benhormann, I think this is a good pattern to automatically support windows via the `certutil` command as fallback you mentioned on the #1535 PR.